src/components/grid/grid.stories.mdx

```diff
@@ -429,11 +429,13 @@ This example is best viewed in the Canvas tab using full-screen mode with device
 
 ## Props
 
-<StyledSystemProps of={GridContainer} spacing />
+### GridContainer Props
```

Comment:
Would you mind updating the stories (think it's just subgrid) that are passing "primary" to the `as`/ `podType` props? They are causing incorrect HTML to be rendered, they should be using `variant` but were likely missed when a refactor was done 🤦 

![image](https://user-images.githubusercontent.com/44157880/176714471-fc8f945b-a44f-430d-afe2-d26abf226732.png)